pkg/pillar/types/zedroutertypes.go

```diff
@@ -1940,6 +1964,14 @@ type NetworkInstanceInfo struct {
 	// to this bridge and their statistics.
 	// We add statistics from all vifs while reporting to cloud.
 	VifMetricMap map[string]NetworkMetric
+
+	// Assigned AppNums on this NetworkInstance
+	AppNumBitMap Bitmap
+	// host IP Addresses needs to managed
+	HostIPAddrModInitialized bool
+	HostIPAddressCount       int
+	ActiveHostIPAddressCount int
+	BridgeAppNum             int
```

Comment:
What is the relationship between this field and the constant AppNumBridge below? 
Do we need both a variable and a constant for it?

Reply:
The field BridgeAppNum can be any IPAddress Map inside the subnet range, and the constant is the default (1st Host IP Address) in the subnet.

Reply:
Wouldn't we want the Bridge to a predictable number in the subnet? Thus we can reserve appNumOnNet==0 for the bridge basically special casing its allocation?